plugins/org.jboss.tools.arquillian.ui/META-INF/MANIFEST.MF

```diff
@@ -30,19 +30,19 @@ Require-Bundle: org.eclipse.ui;bundle-version="3.103.0",
  org.eclipse.jdt.core.manipulation;bundle-version="1.5.0",
  org.eclipse.m2e.core.ui,
  org.eclipse.compare.core,
- org.eclipse.sapphire.platform;bundle-version="0.7.1",
- org.eclipse.sapphire.modeling;bundle-version="0.7.1",
- org.eclipse.sapphire.modeling.xml;bundle-version="0.7.1",
- org.eclipse.sapphire.java;bundle-version="0.7.1",
- org.eclipse.sapphire.ui;bundle-version="0.7.1",
- org.eclipse.sapphire.ui.swt.xml.editor;bundle-version="0.7.1",
+ org.eclipse.sapphire.platform;bundle-version="9.0.0",
```

Comment:
I would use 9.0.0.201505051659 in MANIFEST.MF too. It will help avoid problems if the plugin is installed as a dependency.